### ISSUE TITLE
Chrome no longer supports Flash global settings?

### ISSUE THREAD

**Issue by Accelyte** [KC3Kai/KC3Kai#1515]
Versions:
Chrome version: 52.0.2743.116 m (64-bit)
Windows 10 64-bit (all Windows Updates)
KC3 Kai version 30.8.1
Only tried the webstore version

I'm getting the dreaded CG loop in the strategy room, and I followed the given instructions, but nothing worked...
1. Installed all 3 versions of Flash (PPAPI, Shockwave Player, even NPAPI although I know it doesn't work), no results. Panel remained showing the 'This content requires Flash' warning.
2. Tried disabling and reenabling the Shockwave Player in chrome://plugins, CG was disabled and the error message told me to reenable it. Same result with the panel.
3. Was extremely boned because I was looking forward to Kuma's Summer CG so I tried opening the global settings panel in the Edge browser, and it opened successfully.
4. Figured it might be a Chrome problem instead of a Flash problem so I went on the Google Product Forums, here's what I found:
https://productforums.google.com/forum/#!topic/chrome/zEzA1H7s7fk
https://productforums.google.com/forum/#!topic/chrome/LGt7-YWogPU
Apparently the Global Settings have been disabled for security reasons...am I the only person who's suffering from this?
https://www.dropbox.com/s/lqfhp07r11ws55f/error.jpg

**Comment by dragonjet** [comment]
mine's still ok both strategy room and global settings on macromedia site so it must be system-specific

the best setup would be to just enable the chrome default flash plugin
if that doesnt work, next best thing is to disable the chrome flash, and install the PPAPI version
(when using the NPAPI version kc3 will not be able to detect game data on F12 panel)

having that error message doesnt make sense in the first place since you already have flash installed and can play kancolle.

**Comment by Accelyte** [comment]
Nope, doesn't seem to be working. I even followed Adobe's uninstall guide and cleaned up any system Adobe Flash files...
I've had this issue before, but I didn't report it, but that was on my old system. I reinstalled Windows while moving system drives so I'm sure this copy is clean, yet it's still happening...do you have any other clues as to why it's happening?
Just curious, but will KC3 ever move towards HTML5? I heard Flash will be removed completely by December, so will there be any updates to the visual library?

**Comment by Accelyte** [comment]
I went through the console errors that appeared in Chrome when opening the Global Settings manager. There were a couple of errors, including a "No 'Access-Control-Cross-Origin' header" error. I did a bit of Google research and found this: http://stackoverflow.com/questions/20035101/no-access-control-allow-origin-header-is-present-on-the-requested-resource, with this specific code:
`chrome.exe --user-data-dir="C:/Chrome dev session" --disable-web-security`
I tried this out and it Global Settings manager worked. This confirms my suspicions: Chrome is blocking the Global Settings manager by default. 'web security' huh, I guess I'll have to put myself in danger when using Chrome for KC from now on...
